packages/doc-spa/.env.example

```diff
@@ -1,5 +1,3 @@
-REACT_APP_OPCBASE_API_BASE_PATH=
```

Comment:
Why is this removed. This is used env variable

Reply:
Oops my bad restored it